src-ui/p.html

```diff
@@ -176,9 +176,17 @@ <h2 id="title2">読み込み中です...</h2>
       <div class="child" data-value="clear">__inputmode.clear__</div>
       <div class="child" data-value="info-line">__inputmode.info-line__</div>
       <div class="child" data-value="info-blk">__inputmode.info-blk__</div>
+      <div class="child" data-value="info-ublk">__inputmode.info-ublk__</div>
       <div class="child" data-value="info-road">__inputmode.info-road__</div>
      </div>
     </div>
+    <div class="config" data-config="disptype_context">
```

Comment:
Seems fine to me. I only did it this way because I couldn't figure out how to make it a checkbox. I don't actually know how any of the html or JS/html interaction works, so I just did a copy/paste of yajilin functionality